src/utils/validators.js

```diff
@@ -122,6 +122,11 @@ export function isValidPhoneWithoutCountryCode(phone: string) {
   return re.test(phone);
 }
 
+export function isValidCash(s) {
+  const rgx = /^[0-9]*\.?[0-9]*$/;
```

Comment:
Consider updating the regex
```suggestion
  const regex = /^\d+\.?\d{0,2}$/;
```
At least one digit, optional dot, and at most two digits.

Reply:
Done